bundle/src/main/java/com/adobe/acs/commons/mcp/form/DialogProviderAnnotationProcessor.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2020 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.mcp.form;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.Generated;
+import javax.annotation.processing.AbstractProcessor;
+import javax.annotation.processing.RoundEnvironment;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.tools.JavaFileObject;
+import org.apache.commons.lang.StringUtils;
+import org.apache.sling.models.annotations.Model;
+
+/**
+ * Processes the DialogProvider annotation, producing a corresponding OSGi
+ * service to provide the generated sling dialog resources. This annotation
+ * processor will skip any classes which do not identify their corresponding
+ * sling model either as part of the model annotation or by a property or getter
+ * method.
+ */
+@Generated("Don't run code coverage, JaCoco!")
```

Comment:
Is _this_ class generated? Why not do code coverage?

Reply:
Because it was bombing!   I eventually got Surefire to work after adding the Jacoco runtime dependency but I couldn't get Corbertura to work unless I told it to skip this class.  Funny enough even with this annotation the stupid thing was running code coverage anyway so I should just remove this vestige.